### PR TITLE
[full-ci] update reva to e2232b5a63df

### PR DESCRIPTION
**Pull request by butonic**
update reva to v2.6.2-0.20220706093852-e2232b5a63df

### CODE REVIEW

File: changelog/unreleased/update-reva-beta.5.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Enhancement: Update reva
+
+https://github.com/owncloud/ocis/pull/4114
```
Comment (phil-davis) [inline changelog/unreleased/update-reva-beta.5.md:3]:
```suggestion
https://github.com/owncloud/ocis/pull/4115
```

Comment (butonic) [reply changelog/unreleased/update-reva-beta.5.md]: weird 4114 did not show up in https://github.com/owncloud/ocis/issues?q=

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Core-API-Tests-ocis-storage-10</strong>](https://drone.owncloud.com/owncloud/ocis/13114/50/1) failed. Further test are cancelled...

File: tests/acceptance/expected-failures-webUI-on-OCIS-storage.md [hunk 2]
```diff
@@ -65,8 +65,7 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
 - [webUISharingPublicManagement/publicLinkIndicator.feature:81](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicManagement/publicLinkIndicator.feature#L81)
 - [webUISharingPublicManagement/publicLinkIndicator.feature:98](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicManagement/publicLinkIndicator.feature#L98)
-- [webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:61](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature#L61)
-- [webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:20](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature#L20)
+- [webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature:61](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalGroupsSharingIndicator/shareWithGroups.feature#L20)
```
[the remark this answers is not in the document]
Comment (phil-davis) [reply tests/acceptance/expected-failures-webUI-on-OCIS-storage.md]: I adjusted and pushed a commit.

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/13143/75/14
```
runsh: Total unexpected failed scenarios throughout the test run:
apiWebdavOperations/downloadFile.feature:35
```
That failed again the same after I restarted.

```
  Background:                                                                                   # /drone/src/tests/parallelDeployAcceptance/features/apiWebdavOperations/downloadFile.feature:7
    Given using "oc10" as owncloud selector                                                     # ParallelContext::usingOwncloudSelector()
    And user "Alice" has been created with default attributes and without skeleton files        # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()
    And user "Alice" has uploaded file with content "ownCloud test text file" to "textfile.txt" # FeatureContext::userHasUploadedAFileWithContentTo()

  Scenario: Get the size of a file                                              # /drone/src/tests/parallelDeployAcceptance/features/apiWebdavOperations/downloadFile.feature:35
    Given using "ocis" as owncloud selector                                     # ParallelContext::usingOwncloudSelector()
    When user "Alice" gets the size of file "textfile.txt" using the WebDAV API # FeatureContext::userGetsSizeOfFileUsingTheWebdavApi()
    Then the HTTP status code should be "207"                                   # FeatureContext::thenTheHTTPStatusCodeShouldBe()
      HTTP status code 404 is not the expected value 207
      Failed asserting that 404 matches expected '207'.
    And the size of the file should be "23"                                     # FeatureContext::theSizeOfTheFileShouldBe()
```

Other scenarios that actually download the file are passing. 

`When user "Alice" gets the size of file "textfile.txt" using the WebDAV API` does a PROPFIND with depth 0 on the file. But that is returning a 404 "not found".

Did something change in reva code that might allow GET (download) to work across oC10<->oCIS in parallel, but that PROPFIND does not work?

**Comment by phil-davis** [comment]
I raised issue #4122 and added this to the parallel-deployment expected-failures for now. Everything else passes, so someone could look into the issue separately.

**Comment by phil-davis** [comment]
Codacy complains "[list-item-indent] Incorrect list-item indent: add 2 spaces" - but every other expected-failures file does not have that indent. I am ignoring it.

**Comment by phil-davis** [comment]
More annoying stuff - the expected-failures-file linter from core does not understand where the referenced feature-files should be found. core PR https://github.com/owncloud/core/pull/40184 will fix that.

**Comment by phil-davis** [comment]
I will merge core https://github.com/owncloud/core/pull/40184 and then adjust core commit id...

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
Rebased again! The core commit id got updated just recently in a separate PR, so no need here any more.

**Pull request merged**